index.js

```diff
@@ -219,3 +219,31 @@ window.addEventListener('load', () => {
     });
   });
 });
+
+const email = document.getElementById('email');
+
+document.querySelector('#contact_button').addEventListener('click', (e) => {
+  const name = document.getElementById('name');
+  if (!name.validity.valid) {
+    document.getElementById('alertmessage').innerHTML = 'Name field is empty!';
+    e.preventDefault();
+    return;
+  }
+  if (email.validity.typeMismatch) {
+    document.getElementById('alertmessage').innerHTML = 'Email field is Empty!';
+    e.preventDefault();
+    return;
```

Comment:
- [ ] Kindly make sure the form cannot submit when the email is not fully in lower case, as requested in the project requirements [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=The%20content%20of%20the%20email%20field%20has%20to%20be%20in%20lower%20case.) and must deplay "Email must be in lowercase